person.rb

```diff
@@ -24,6 +25,11 @@ def correct_name
     name
   end
 
+  def add_rental(date, book)
+    rental = Rental.new(date, book, self)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] If you look at your Rental class you would see that creating a rental also adds the rental to the respective `:rentals` of book and person. So we need to remove line 30.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```